# Theme of this notebook - How to calculate probabilities of sample distributions
* Let's practice throughout solving sample questions

# Probrem #1
A simple random sample of size n = 49 is obtained from a population with μ = 80 and σ = 14.

# (a) Describe the sampling distribution of x?

The sampling distribution of x̅:

Mean:
$μ_{x̅}$ = μ = 80<br>
Standard Deviation: $σ_{x̅}$ = $\frac{σ}{\sqrt{n}}$ = $\frac{14}{\sqrt{49}}$ = 2<br>

Therefore, the sampling distribution is: x̅ ~ N(80,2)

In [15]:
# (b) What is P(x > 83)

#Import libraries
import numpy as np
from scipy.stats import norm

#Define each variable
mu_p1 = 80
sigma_p1 = 14/np.sqrt(49)
zscore_p1_b = (83 - mu_p1) / sigma_p1
prob_p1_b = 1- norm.cdf(zscore_p1_b)
print(f'Probability: {prob_p1_b*100}%')

Probability: 6.6807201268858085%


In [16]:
# (c) What is P(x <= 75.8)
zscore_p1_c = (75.8 - mu_p1) / sigma_p1
prob_p1_c = norm.cdf(zscore_p1_c)
print(f'Probability: {prob_p1_c*100}%')

Probability: 1.7864420562816494%


In [17]:
# (d) What is P(78.3<x<85.1)
zscore_p1_d_right = (85.1 - mu_p1) / sigma_p1
zscore_p1_d_left = (78.3 - mu_p1) / sigma_p1
prob_p1_d = norm.cdf(zscore_p1_d_right) - norm.cdf(zscore_p1_d_left)
print(f'Probability: {prob_p1_d*100}%')

Probability: 79.69513109232413%


# Problem #2:
The reading speed of second grade students is approximately normal, with a mean of 80 words per minute (wpm) and a standard deviation of 10 wpm.

In [18]:
# Problem (a)
# What is the probability a randomly selected student
# will read more than 95 words per minute?

#Define each variable
mu_p2_a = 90
sigma_p2_a = 10
zscore_p2_a = (95 - mu_p2_a) / sigma_p2_a
prob_p2_a = 1- norm.cdf(zscore_p2_a)
print(f'Probability: {prob_p2_a*100}%')

Probability: 30.853753872598688%


In [19]:
# Problem (b)
# What is the probability that a random sample of 12 second grade students
# results in a mean reading rate of more than 95 words per minute?

#Define each variable
mu_p2_b = 90
sigma_p2_b = 10/np.sqrt(12)
zscore_p2_b = (95 - mu_p2_b) / sigma_p2_b
prob_p2_b = 1- norm.cdf(zscore_p2_b)
print(f'Probability: {prob_p2_b*100}%')

Probability: 4.16322583317752%


In [20]:
# Problem (c)
# What is the probability that a random sample of 24 second grade students
# results in a mean reading rate of more than 95 words per minute?

# Define each variable
mu_p2_c = 90
sigma_p2_c = 10/np.sqrt(24)
zscore_p2_c = (95 - mu_p2_c) / sigma_p2_c
prob_p2_c = 1- norm.cdf(zscore_p2_c)
print(f'Probability: {prob_p2_c*100}%')

Probability: 0.7152939217714871%


# Problem (d)
What effect does increasing the sample size have on the probability? Provide an explanation for this result.

My Answer:<br>
The more the sample sizes increase, the lower probability that the mean reading speed of second grade students is more than 95 words per minute become.<br>
This is because the standard deviation become lower as the sample sizes increase, so more second grade students' reading speeds converge to the mean.

In [21]:
# Problem (e)
# A teacher instituted a new reading program at school.
# After 10 weeks in the program, it was found that the mean reading speed of
# a ramdom sample of 20 second grade students was 92.8 wpm.
# What might you conclude based on this result?

# Define each variable
mu_p2_e = 90
sigma_p2_e = 10/np.sqrt(20)
zscore_p2_e = (92.8 - mu_p2_e) / sigma_p2_e
prob_p2_e = 1- norm.cdf(zscore_p2_e)
print(f'Probability: {prob_p2_e*100}%')

Probability: 10.524885002766894%


Based on the result, we conclude that about 10.52% of samples will be above 92.8 wpm.

# Problem #3
A simple random sample of size n = 1,000 is obtained from a population whose size is N = 1,000,000 and whose population proportion with a specified characteristic is p = 0.35

# (a) Describe the sampling distribution of p̂?

The sampling distribution of p̂:

Mean:
$μ_{p̂}$ = μ = 0.35<br>
Standard Deviation: $σ_{p̂}$  =
$\frac{\sqrt{p(1-p)}}{\sqrt{1000}}$ = $\sqrt{0.35×(1-0.35)}$ ÷ ${10\sqrt{10}}$ = 0.01508...<br>

Therefore, the sampling distribution is: p̂ ~ N(0.35, 0.015)

In [22]:
# Problem (b)
# What is the probability of obtaining x = 390 or more indiduals
# with the characteristic?

# Define each variable
mu_p3 = 1000 * 0.35
sigma_p3 = 1000 * np.sqrt(0.35*0.65) / np.sqrt(1000)
zscore_p3_b = (390-mu_p3) / sigma_p3
prob_p3_b = 1- norm.cdf(zscore_p3_b)
print(f'Probability: {prob_p3_b*100}%')

Probability: 0.40011330941790435%


In [23]:
# Problem (c)
# What is the probability of obtaining x = 320 or fewer indiduals
# with the characteristic?

# Define each variable
zscore_p3_c = (320-mu_p3) / sigma_p3
prob_p3_c =norm.cdf(zscore_p3_c)
print(f'Probability: {prob_p3_c*100}%')

Probability: 2.3351670374665328%


# Problem #4
According to a study conducted by the Gallup organization, the proportion of Americans who are satisfied with the way things are going in their lives is 0.82.<br>
Suppose that a random sample of 100 Americans is obtained.

# (a) Describe the sampling distribution of p̂?

The sampling distribution of p̂:

Mean:
$μ_{p̂}$ = μ = 0.82<br>
Standard Deviation: $σ_{p̂}$ =
$\frac{\sqrt{p(1-p)}}{\sqrt{100}}$ = $\sqrt{0.82×(1-0.82)}$ ÷ 10 = 0.03841...<br>

Therefore, the sampling distribution is: p̂ ~ N(0.82, 0.038)

In [24]:
# Problem (b)
# What is the probability that at least 85 Americans in the sample are
# sample are satisfied with their lives?
mean_p4 = 100 * 0.82
sigma_p4 = 100 * np.sqrt(0.82*0.18) /10
zscore_p4_b = (85-mean_p4)/sigma_p4
prob_p4_b = 1- norm.cdf(zscore_p4_b)
print(f'Probability: {prob_p4_b*100}%')

Probability: 21.74398292478916%


In [25]:
# Probrem (c)
# What is the probability that 75 or fewer Americans in the sample are
# satisfied with their lives? Is this result unusual?
zscore_p4_c = (75-mean_p4)/sigma_p4
prob_p4_c = norm.cdf(zscore_p4_c)
print(f'Probability: {prob_p4_c*100}%')

# Is this result unusual?
# Yes, this result is unusual becuase the probability is less than 4%.

Probability: 3.4225427637043824%


# Problem #5
According to a USA Today "Snapshot", 26% of adults do not have any credit cards. A simple random sample of 500 adults is obtained.

# Problem (a)
# Describe the sampling distribution of p̂, the sample proportion of adults who do not have a credit card.

The sampling distribution of p̂:

Mean:
$μ_{p̂}$ = μ = 0.26<br>
Standard Deviation: $σ_{p̂}$ =
$\frac{\sqrt{p(1-p)}}{\sqrt{500}}$ = $\sqrt{0.26×(1-0.26)}$ ÷ 10$\sqrt{5}$ = 0.01961...<br>

Therefore, the sampling distribution is: p̂ ~ N(0.26, 0.020)

In [26]:
# Problem(b)
# In a random sample of 500 adults, what is the probability that less
# than 24% have no credit cards?
mean_p5 = 500 * 0.26
sigma_p5 = 500 * np.sqrt(0.26*0.74) / np.sqrt(500)
zscore_p5_b = (500*0.24-mean_p5)/sigma_p5
prob_p5_b = norm.cdf(zscore_p5_b)
print(f'Probability: {prob_p5_b*100}%')

Probability: 15.396877265468456%


In [27]:
# Problem(c)
# Would it be unusual if a random sample of 500 adults results in
# 150 having no credit cards?
zscore_p5_c = (150-mean_p5)/sigma_p5
prob_p5_c = 1 - norm.cdf(zscore_p5_c)
print(f'Probability: {prob_p5_c*100}%')

# Is this result unusual?
# Yes, this result is unusual becuase the probability is approximately 2%.

Probability: 2.0719102343641826%


In [14]:
#---End---